<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Map</class>
 <widget class="QWidget" name="Map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>193</width>
     <height>155</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_Name"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Hp"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Mp"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Sp"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Act"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit_Skill">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>301</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Item">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>301</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>战斗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addItem">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加物品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addSkill">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加技能</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
